### PR TITLE
VAULT-29013 Read a Vault Secrets Integration

### PR DESCRIPTION
**Pull request by mercedesbh**
### Changes proposed in this PR:
This PR adds the read command for hvs integrations for ticket https://hashicorp.atlassian.net/browse/VAULT-29013

### How I've tested this PR:
Ran `make go/build` to manually test local changes, and added unit tests.

### How I expect reviewers to test this PR:
1. Checkout this branch
2. Run `make go/build`
3. Read an integration `./bin/hcp vault-secrets integrations read {YOUR_INTEGRATION_NAME} --type {YOUR_INTEGRATION_TYPE}`

### Output of affected commands:
```
❯ ./bin/hcp vs integrations --help                                                                                                                                                                           ─╯
USAGE
  hcp vault-secrets integrations <command> [Optional Flags]

DESCRIPTION
  The hcp vault-secrets integrations command group lets you manage Vault Secrets
  integrations.

COMMANDS
  read:  Read a Vault Secrets integration.

❯ ./bin/hcp vs integrations read --help                                                                                                                                                                      ─╯
USAGE
  hcp vault-secrets integrations read NAME --type=TYPE [Optional Flags]

DESCRIPTION
  The hcp vault-secrets integrations read command gets a Vault Secrets
  integration.

EXAMPLES
  Read an integration:
  
  $ hcp vault-secrets integrations read sample-integration --type twilio

POSITIONAL ARGUMENTS
  NAME
    The name of the integration to read.

REQUIRED FLAGS
  --type=TYPE
    The type of the integration to read.

GLOBAL FLAGS
  --debug, --format=FORMAT, --profile=NAME, --project=ID, --quiet
  
  For more global flag details, run $ hcp --help

❯ ./bin/hcp vs integrations read Twil-Int --type twilio                                                                                                                                                      ─╯
Integration Name   Account SID                       
Twil-Int           {TWILIO_SID}
```

### Checklist:
- [x] Tests added if applicable
- [ ] CHANGELOG entry added or label 'pr/no-changelog' added to PR
  > Run `CHANGELOG_PR=<PR number> make changelog/new-entry` for guidance
  > in authoring a changelog entry, and commit the resulting file, which should
  > have a name matching your PR number. Entries should use imperative present
  > tense (e.g. Add support for...)

### CODE REVIEW

File: internal/commands/vaultsecrets/integrations/displayer.go [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+}
+
+func (d *displayer) FieldTemplates() []format.Field {
+	return []format.Field{
+		{
+			Name:        "Integration Name",
+			ValueFormat: "{{ .IntegrationName }}",
+		},
+		{
+			Name:        "Account SID",
```
Comment (pmmukh) [inline internal/commands/vaultsecrets/integrations/displayer.go:75]: Do we also need type aware implementations of the FieldTemplates function? Cause I see there's Twilio details here, wondering if in the case of Mongo we might want other info displayed

**Review by pmmukh — COMMENTED** [review]
couple questions, looks great!

File: internal/commands/vaultsecrets/integrations/displayer.go (hunk 1 above)
Comment (mercedesbh) [reply internal/commands/vaultsecrets/integrations/displayer.go:75]: Good catch! I guess I didn't push up the commit with that addition 🤔. One sec

**Review by pmmukh — APPROVED** [review]
looks great!

**Pull request merged**